.github/workflows/platformDeploy.yml

```diff
@@ -179,6 +179,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
 
+      - name: Archive desktop sourcemaps
+        uses: actions/upload-artifact@v3
```

Comment:
Not sure if we should use v3 here, as for every action we are using upload-artifact v4